agreement.md

```diff
@@ -95,29 +86,29 @@ found it. If you are unsure, ask your team leadership.
 - Exits and walkways must be kept clear of furniture or workpieces for safe passage during an emergency.
 - Never modify any tool, and especially, never remove safety devices from tools.
 - If you see someone working in an unsafe manner, you must gently correct them as a colleague. If you feel uncomfortable
-doing so, contact your team leadership or Dr. Cunefare and inform them of the situation.
+doing so, contact your team leadership, Jacob Tompkins, or Dr. Mayor, and inform them of the situation.
```

Comment:
Would it be any better to refer to these individuals by their position (whatever titles Jake and Dr. Mayor hold w/r/t RJ/SCC) rather than by name or at least include them? This may help newer members understand _why_ these people are specifically called out here rather than random names with no association. For example, `Jacob Tomkins (SCC Manager), Dr. Mayor (???)`.

Reply:
Added